src/main/java/net/fabricmc/loom/api/LoomGradleExtensionAPI.java

```diff
@@ -132,6 +134,18 @@ default void interfaceInjection(Action<InterfaceInjectionExtensionAPI> action) {
 
 	InterfaceInjectionExtensionAPI getInterfaceInjection();
 
+	default void versionsManifests(Action<VersionsManifestsAPI> action) {
+		action.execute(getVersionsManifests());
+	}
+
+	ManifestLocations getVersionsManifests();
+
+	default void experimentalVersionsManifests(Action<VersionsManifestsAPI> action) {
+		action.execute(getExperimentalVersionsManifests());
+	}
+
+	ManifestLocations getExperimentalVersionsManifests();
```

Comment:
Why is the experimental one needed? Couldnt these just be one, with the experimental one coming after the default manifest?

Reply:
I made some changes to combine them, let me know if this is what you had in mind